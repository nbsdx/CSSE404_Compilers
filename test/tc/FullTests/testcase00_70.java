class Test {
    public static void main(String[] args) {
	Foo bar = new Foo();
    }
}

class Foo {
    int avar;
    int anothervar;
}

class Foo2 extends Foo {
    int avar; //should break
}
