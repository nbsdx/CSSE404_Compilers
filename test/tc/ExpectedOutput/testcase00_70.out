The class variable avar is already declared.  Redeclaration and shadowing are not allowed.
